### PR TITLE
Update piped add cloud provider manual docs

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

Change text placeholder in docs.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs/operator-manual/piped/adding-a-cloud-provider.md [hunk 1]
```diff
@@ -69,8 +69,8 @@ spec:
       type: CLOUDRUN
       config:
         project: gcp-project
```
Comment (nghialv) [inline docs/content/en/docs/operator-manual/piped/adding-a-cloud-provider.md:71]: This too.

File: docs/content/en/docs/operator-manual/piped/adding-a-cloud-provider.md [hunk 2]
```diff
@@ -69,8 +69,8 @@ spec:
...
-        region: cloudrun-region
-        credentialsFile: path-to-the-service-account-file
+        region: {CLOUDRUN_REGION}
```
Comment (nakabonne) [inline docs/content/en/docs/operator-manual/piped/adding-a-cloud-provider.md:72]: I would prefer `YOUR_CLOUDRUN_REGION`, but up to you

Comment (nghialv) [reply docs/content/en/docs/operator-manual/piped/adding-a-cloud-provider.md:72]: To be honest, I don't like that `YOUR` prefix. By adding it we will have `YOUR_REGION`, `YOUR_PROJECT`,  `YOUR_CREDENTIAL_FILE`...

Comment (khanhtc1202) [reply docs/content/en/docs/operator-manual/piped/adding-a-cloud-provider.md:72]: 🤣 I have the same sense as @nghialv

Comment (nakabonne) [reply docs/content/en/docs/operator-manual/piped/adding-a-cloud-provider.md:72]: I hear you. `{}` and capitalization are enough representing they are placeholders 👍

File: docs/content/en/docs/operator-manual/piped/adding-a-cloud-provider.md (hunk 1 above)
Comment (khanhtc1202) [reply docs/content/en/docs/operator-manual/piped/adding-a-cloud-provider.md]: addressed on https://github.com/pipe-cd/pipe/pull/2078/commits/a251c4e123c5a21841ab337c1fe8f1d9a0c3ad19 🙏

**Pull request merged**